lib/Github/Api/User.php

```diff
@@ -44,6 +44,18 @@ public function all($id = null)
     }
 
     /**
+     * [UNDOCUMENTED] Get extended information about a user by its ID.
```

Comment:
there are loads of things like this everywhere though, that we don't incode, like repos by id